documento-in-consultazione/il-tavolo-di-lavoro.rst

```diff
@@ -382,7 +382,7 @@ Tabella 3: elenco documenti acquisiti
 +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------+------------------+
-| Improving the User Story Agile Technique Using the INVEST Criteria (23° International Workshop on Software Measurement (IWSM) and 8th International Conference on Software Process and Product Measurement (MENSURA))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Luigi Buglione - Alain Abran                                                                                   | 23/10/2013       |
+| Improving the User Story Agile Technique Using the INVEST Criteria (23º International Workshop on Software Measurement (IWSM) and 8th International Conference on Software Process and Product Measurement (MENSURA))                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Luigi Buglione - Alain Abran                                                                                   | 23/10/2013       |
```

Comment:
Il simbolo di [grado](https://it.wikipedia.org/wiki/Grado_(simbolo)) (°) è stato sostituito con l'[indicatore ordinale](https://it.wikipedia.org/wiki/Indicatore_ordinale) (º), ma sembra qui ammissibile anche l'ordinale inglese "23rd" per uniformità con "8th".